README.md

```diff
@@ -2,25 +2,9 @@
     <a href="http://sylius.com" target="_blank">
         <img src="https://demo.sylius.com/assets/shop/img/logo.png" />
     </a>
-    <br />
-    <a href="https://packagist.org/packages/sylius/sylius" title="License" target="_blank">
-        <img src="https://img.shields.io/packagist/l/Sylius/Sylius.svg" />
-    </a>
-    <a href="https://packagist.org/packages/sylius/sylius" title="Version" target="_blank">
-        <img src="https://img.shields.io/packagist/v/Sylius/Sylius.svg" />
-    </a>
-    <a href="http://travis-ci.org/Sylius/Sylius" title="Build status" target="_blank">
-        <img src="https://img.shields.io/travis/Sylius/Sylius/master.svg" />
-    </a>
-    <a href="https://scrutinizer-ci.com/g/Sylius/Sylius/" title="Scrutinizer" target="_blank">
-        <img src="https://img.shields.io/scrutinizer/g/Sylius/Sylius.svg" />
-    </a>
```

Comment:
Maybe they would work if we use Markdown for them?

[![Build Status](https://travis-ci.org/Sylius/Sylius.svg?branch=master)](https://travis-ci.org/Sylius/Sylius)
[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/Sylius/Sylius/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/Sylius/Sylius/?branch=master)

```
[![Build Status](https://travis-ci.org/Sylius/Sylius.svg?branch=master)](https://travis-ci.org/Sylius/Sylius)
[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/Sylius/Sylius/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/Sylius/Sylius/?branch=master)
```
I agree it's frustrating to see them failing all the time, but, when working, they look nice and it's an important info about project at the beginning of README :)

Reply:
It generates the same HTML (but without `target="_blank"`).